src/main/kotlin/solve/scene/view/LineView.kt

```diff
@@ -32,22 +33,18 @@ class LineView(
         node.endY = finishCoordinates.second
     }
 
-    private var width = OrdinaryWidth
-
     private fun createShape(): Line {
         val shape =
             Line(startCoordinates.first, startCoordinates.second, finishCoordinates.first, finishCoordinates.second)
-        shape.fill = line.layerSettings.getColor(line)
+        shape.stroke = line.layerSettings.color
         shape.opacity = line.layerSettings.opacity
         shape.strokeWidth = width
         return shape
     }
 
     override fun highlightShape() {
-        TODO("Not yet implemented")
```

Comment:
what about impl?

Reply:
That is the next task "Associate lines". Now I removed TODO to avoid exceptions